src/lint/track_config.nim

```diff
@@ -293,7 +293,16 @@ type
 
   Concepts* = seq[Concept]
 
+  FilePatterns* = object
+    solution*: seq[string]
+    test*: seq[string]
+    example*: seq[string]
+    exemplar*: seq[string]
+    editor*: seq[string]
+
   TrackConfig* = object
+    slug*: string
```

Comment:
I added the slug to allow for the `d` and `plsql` tracks to be exceptional cases.